packages/backend/src/connector/core/middleware/balance.ts

```diff
@@ -52,9 +52,9 @@ export function createBalanceMiddleware(): ILPMiddleware {
           throw new InsufficientLiquidityError(trxOrError)
         case TransferError.ReceiveLimitExceeded: {
           const receivedAmount = destinationAmountOrError.toString()
-          assert.ok(accounts.outgoing.id)
-          const receiveLimit = await services.accounts.getReceiveLimit(
-            accounts.outgoing.id
+          assert.ok(accounts.outgoing.receivedAccountId)
+          const receiveLimit = await services.accounts.getBalance(
+            accounts.outgoing.receivedAccountId
```

Comment:
The naming of `receivedAccountId` confuses me. It isn't the amount that has been received, it is the maximum amount that _can_ be received, right?

Reply:
It's confusing because it depends:
https://github.com/interledger/rafiki/pull/197/files#diff-e5a9751319994ef97d1413f0c3bc6587f756163958ad21f877a5fe7d84a08661R81-R89

I considered having separate `receivedAccountId` and `receiveLimitAccountId`, but ~`transferFunds`~`sendAndReceive` would treat them the same.